docs/modules/ROOT/pages/config_properties.adoc

```diff
@@ -1,58 +1,105 @@
+The following listing shows a fairly complete example that sets most available options, for more detail about the individual elements please refer to the table further down on the page.
 
 [source,yaml]
 ----
 apiVersion: nifi.stackable.tech/v1alpha1
 kind: NifiCluster
 metadata:
-  name: simple
+  name: simple-nifi
 spec:
-  metricsPort: 8428
-  version: "1.13.2"
+  version: "1.15.0"
   zookeeperReference:
-    name: simple
+    name: simple-zk
     namespace: default
     chroot: /nifi
+  authenticationConfig:
+    method:
+      SingleUser:
+        adminCredentialsSecret:
+          name: nifi-admin-credentials-simple
+          namespace: default
+    allowAnonymousAccess: true
   nodes:
     roleGroups:
       default:
         selector:
           matchLabels:
             kubernetes.io/os: linux
         config:
-          httpPort: 10000
-          protocolPort: 10443
-          loadBalancePort: 6342
+          sensitivePropertyKeySecret: nifi-sensitive-property-key
+          log:
+            rootLogLevel: INFO
+        replicas: 3
 ----
+=== Node Configuration
+[source,yaml]
+----
+ nodes:
+    roleGroups:
+      default:
+        selector:
+          matchLabels:
+            kubernetes.io/os: linux
+        config:
+        replicas: 3
+----
+The `nodes` element is used to define how many pods with which configuration should be rolled out.
+It is possible to define multiple groups of nodes, each with its own distinct configuration, every `roleGroup` has the following elements:
+
+ - selector: a Kubernetes `Selector` to specify criteria that can be used to target nodes
+ - config: The NiFi config to use for this group
+ - replicas: How many pods to roll out for this group
+
+=== Authentication
+[source,yaml]
+----
+  authenticationConfig:
+    method:
+      SingleUser:
+        adminCredentialsSecret:
+          name: nifi-admin-credentials-simple
+          namespace: default
+    allowAnonymousAccess: true
+----
+All authentication related parameters are configured in the authenticationConfig element.
 
+==== Authentication Method
+Currently the only supported authentication method is "SingleUser", which allows the definition of one admin user which can then access the cluster.
+Specification of these users credentials happens via referring to a Secret in Kubernetes, this secret will need to contain at least the two keys `username` and `password`.
+Extra keys may be present, but will be ignored by the operator.
 
-[cols="1,1,1,1,1"]
-|===
-| config-spec Property | Actual Property | Default Value | Recommended Value | Location
+The value for the `password` key has to be a bcrpyted (weight=12) password that will be used for the admin user.
+Hashing of the password can for example be done via https://bcrypt.online/ or with various command line tools, the https://bcrypt.online/[NiFi admin guide] has additional information on this hashing method.
```

Comment:
```suggestion
Hashing of the password can for example be done via https://bcrypt.online/ or with various command line tools, the https://nifi.apache.org/docs/nifi-docs/html/administration-guide.html#bcrypt-kdf[NiFi admin guide] has additional information on this hashing method.
```

Reply:
apologies, that doc was outdated, we do not take a pre-hashed password anymore but do the hashing ourselves.